code/controllers/subsystem/disease.dm

```diff
@@ -12,6 +12,13 @@ SUBSYSTEM_DEF(disease)
 	if(!diseases)
 		diseases = subtypesof(/datum/disease)
 
+/datum/controller/subsystem/disease/Initialize(timeofday)
+	var/list/all_common_diseases = diseases - typesof(/datum/disease/advance)
```

Comment:
`subtypesof` maybe?

Reply:
No, since we want to exclude the advanced disease (aka everything made in viro except from flu preset)